package.json

```diff
@@ -79,10 +74,6 @@
 					"command": "metaed.build",
 					"group": "navigation"
 				},
-				{
-					"command": "metaed.about",
-					"group": "navigation"
-				},
```

Comment:
Why remove the About button?

Reply:
The "About" function is handled by the readme.md being the extension "home" page. The only other thing it did was listed the installed plugins, but you can get that from the Output window.